### PR TITLE
Clearing up existing JSHint errors before I go and add some new ones :)

### PR DESCRIPTION
**Pull request by urfolomeus**

### CODE REVIEW

**Comment by munckymagik** [comment]
:+1:

File: assets/js/lib/mediaQueries.js [hunk 1]
```diff
@@ -21,6 +21,18 @@ define(['jquery', 'eventsWithPromises', 'featureDetect', 'jqueryThrottleDebounce
...
+   * Media Queries with Sass](http://davidwalsh.name/device-state-detection-css-media-queries-javascript)
+   * @function
+   * @return {String} mq-xs or mq-s or mq-m or mq-l or mq-xl
+   */
+  function getSize() {
+    return featureDetect.mediaQueries ?
+      window.getComputedStyle(testElement[0], ':after').getPropertyValue('content') : 'mq-l';
+  }
+
+  /**
```
Comment (slaywell) [inline assets/js/lib/mediaQueries.js]: Is this just reordering the functions? Out of interest, was this causing JSHint errors?

Comment (urfolomeus) [inline assets/js/lib/mediaQueries.js]: Yeah it was complaining that `getSize()` was being called before it was defined.

**Comment by slaywell** [comment]
:+1:

**Comment by pmcostello** [comment]
:+1:

**Pull request merged**